src/frontend/Komponenter/Personoversikt/Dokumenter.tsx

```diff
@@ -49,37 +57,75 @@ const Dokumenter: React.FC<{ personopplysninger: IPersonopplysninger }> = ({
+                const grupperteDokumenter = groupBy(dokumentResponse, (i) => i.journalpostId);
+
                 return (
                     <>
                         <DokumenterVisning>
                             <TabellWrapper>
                                 <TabellOverskrift Ikon={Mappe} tittel={'Dokumenter'} />
                                 <table className="tabell">
                                     <thead>
-                                        <Kolonnetittel text={'Dato'} width={35} />
-                                        <Kolonnetittel text={'Tittel'} width={35} />
+                                        <Kolonnetittel text={'Dato'} width={15} />
+                                        <Kolonnetittel text={'Tittel'} width={75} />
+                                        <Kolonnetittel text={'Status'} width={10} />
                                     </thead>
                                     <tbody>
-                                        {dokumentResponse.map(
-                                            (dokument: Dokumentinfo, indeks: number) => {
-                                                return (
-                                                    <tr key={indeks}>
-                                                        <Td>
-                                                            {formaterNullableIsoDatoTid(
-                                                                dokument.dato
-                                                            )}
-                                                        </Td>
-                                                        <Td>
-                                                            <Lenke
-                                                                onClick={() =>
-                                                                    hentDokument(dokument)
-                                                                }
-                                                                href={'#'}
-                                                            >
-                                                                {dokument.tittel}
-                                                            </Lenke>
-                                                        </Td>
-                                                    </tr>
+                                        {Object.keys(grupperteDokumenter).map(
+                                            (journalpostId: string) => {
+                                                return grupperteDokumenter[journalpostId].map(
+                                                    (dokument: Dokumentinfo, indeks: number) => {
+                                                        if (indeks === 0) {
+                                                            return (
+                                                                <TrHoveddokument key={indeks}>
```

Comment:
Får vi duplikate keys nå? Må vi ha `key={journalpostId+"-"indeks}` for å unngå duplikate keys?

Reply:
Bra catch, fikset i 06b8d38